src/components/error.js

```
@@ -0,0 +1,24 @@
+import React from "react";
+
+const Error = ({ error }) => {
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)